roles/common/tasks/get_authorization_token.yml

```diff
@@ -27,7 +27,6 @@
   delay: 5
   no_log: "{{mask_secrets|bool}}"
   when:
-    - cluster_id_source | default('erp') == 'erp'
```

Comment:
Why is this condition removed now

Reply:
noticed it was a mistake, this is idp authorization and does not depend on erp. MDS auth depends on erp

Reply:
okay makes sense